### ISSUE TITLE
authorization - changes of permissions aren't reflected until user logs off/on again

### ISSUE THREAD

**Issue by tonysurma** [HTBox/allReady#1876]
set a user as org admin and had to log in/off to get perms change - should be reflected immediately ideally

**Comment by chris-sargood** [comment]
Picking this one up

**Comment by chris-sargood** [comment]
@stevejgordon 

Will this be solved by the middleware you were showing me earlier?

**Comment by stevejgordon** [comment]
@chris-sargood - Yes at some point it should do. I think maybe we park this behind what I'm doing.

**Comment by chris-sargood** [comment]
@stevejgordon 

Okay leaving this one to you.

**Comment by EmilyLuijbregts** [comment]
@stevejgordon - any update on this ticket?

**Comment by stevejgordon** [comment]
@EmilyLuijbregts I don't think this is solved yet. We have a larger piece of work around possibly implement Identity Server to support to SSO requirements so might wait until that is defined as this can then be part of that work. I'll try to load up the site and double check if this is still an issue though, and if there's a short term fix we can add quite easily.
